airflow/sensors/external_task.py

```diff
@@ -217,6 +226,13 @@ def poke(self, context, session=None):
                 serialized_dttm_filter,
             )
 
+        if self.external_dag_id and not self.external_task_group_id and not self.external_task_ids:
+            self.log.info(
+                "Poking for dag '%s' on %s ... ",
```

Comment:
```suggestion
                "Poking for DAG '%s' on %s ... ",
```

I wonder if this message could be spammy? This would appear again and again until the poke is successful.

Reply:
for us this message is very helpful to know which run_id we are pocking without scrolling around when we are polling for the DAG status.

Reply:
OK. I don’t feel too strong about this, and we can always revisit if someone complains.